packages/runner/src/main.js

```diff
@@ -33,9 +72,11 @@ function createWindow () {
 
   ipcMain.on('study.load', (e, filePaths) => {
     console.log('loading file paths from', filePaths)
-    const studyWindow = new StudyWindow(filePaths, {
-      development: inDevelopment,
-    })
+    // const studyWindow = new StudyWindow(filePaths, {
```

Comment:
I'm not sure I understand why this code is commented out -- what's the goal here? If there's no explicit aim, I would encourage you to remove the changes in the `study.load` IPC handler.

Reply:
Sure , I am removing this code here. I will be adding the IPC handler code in the JSON PR.